core/outbox_test.go

```diff
@@ -129,22 +135,51 @@ func TestOutbox(t *testing.T) {
 
 		ob := core.NewOutbox(w, nullValidator{}, queue, publisher, nullPolicy{}, provider, provider)
 
-		_, err := ob.Send(context.Background(), sender, toAddr, types.ZeroAttoFIL, types.NewGasPrice(0), types.NewGasUnits(0), "")
+		_, err := ob.Send(context.Background(), sender, toAddr, types.ZeroAttoFIL, types.NewGasPrice(0), types.NewGasUnits(0), true, "")
 		assert.Error(t, err)
 		assert.Contains(t, err.Error(), "account or empty")
 	})
+	t.Run("enqueues and does not bcast if bcast set to false", func(t *testing.T) {
```

Comment:
You could cut down on the test code by combining this into the test at line 40, and looping over a `bcast` value in the array `{true, false}`.